app/graphql/github.js

```diff
@@ -0,0 +1,64 @@
+import {isIterable} from '../utils/commons';
+
+export const tasksQuery = `query($searchQuery: String!){
+  rateLimit {
+    remaining
```

Comment:
Is this being used or going to be used somewhere?

Reply:
@bekicot

Reply:
Currently no use

Reply:
Please remove, so it is explicitly added in the correct PR when it is going to be used.